templates/taxi/driver_list.html

```diff
@@ -1,30 +1,41 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
```

Comment:
Ensure that the URL for creating a new driver is correctly set up in the `urls.py` file and that the corresponding view handles the creation process.